### DISCUSSION TITLE
Mountain Duck - Slow transfers

### DISCUSSION THREAD

**Discussion by thebeq** [iterate-ch/cyberduck#16273] [Q&A]
I am mapping a LAN SFTP drive with Mountain Duck (MD).

I get about 4.5MiB/s which is very slow. Using a standard SFTP client, I can get 33MiB/s.

Is there any way to diagnose the issue or possibly improve it?

**Comment by tatze96** [comment]
We are aware of slow transfer speeds for SFTP but haven't found a way to improve them within the current SFTP implementation yet.

Could you send us the following information so we can compare the present cases?
- Which operating system is in use?
- Which connection mode (*Smart Synchronization*, or *Online*) is in use?
- Is upload, download or both affected?
